kubernetes_env/benchmark/benchmark.py

```diff
@@ -226,7 +309,7 @@ def main(args):
     parser.add_argument("-fo",
                         "--fortio",
                         dest="fortio",
-                        type=bool,
+                        default=True,
```

Comment:
This is a little counter intuitive. I would use a different flag if you want to use a custom load generator.  E.g, --use-custom or something like that.